requestor/yagna_erigon_manager.py

```diff
@@ -5,23 +5,51 @@
 from yapapi import Executor, Task
 from datetime import timedelta
 from worker import worker
+from dataclasses import dataclass, field
+from typing import Optional
+from datetime import datetime
+
 
 SUBNET_TAG = 'ttt'
 
 
+@dataclass
+class RuntimeState():
+    status: str
+    url: Optional[str] = None
+    secret: Optional[str] = None
+    timestamp: datetime = field(init=False)
+
+    def __post_init__(self):
+        self.timestamp = datetime.now()
+
+
 class Erigon():
     def __init__(self):
         self.id = self._create_id()
         self.queue = asyncio.Queue()
         self.start_fut = None
         self.stopped = False
+        self.runtime = RuntimeState('initializing')
```

Comment:
I'm pretty sure this field should not be called `runtime` but `state`.

Reply:
Btw, how about prepending all the 'private' fields with and underscore? I think `queue`, `start_fut`, and `update_task` should not be externally accessed.

Reply:
1. `runtime` vs `state`

We could have a `state` of Erigon object, e.g. "cancelled by owner" or "could not connect to yagna requestor" etc.
I gave this name to distinguish those requestor-side states from something that strictly refers to provider-side only.
I will change this doesn't convince you : )

2. `queue` is accessed in `worker.py'. `update_task` probably *should* be accessed by some part of the code that performs a cleanup after stopping. In general, I'm 100% yes about the underscores but I would add this later, when we have a final interface.

Reply:
Ad. 1 Let's call it `runtime_state` then.
Ad. 2 Fair enough.

Reply:
Maybe `erigon.state.runtime`?

Reply:
If I understand your proposal correctly, that would require introducing some intermediary data structure (`erigon.state`) but it's okay for me.

Reply:
It is `runtime_state`.